### PR TITLE
 [Docathon] Fix NO.1-NO.5 API label

### PR DESCRIPTION
**Pull request by enkilee**
### PR types

Others
### PR changes

Docs
### Description

* https://github.com/PaddlePaddle/Paddle/issues/57363
No.1 to No.5
No.1 : OK
No.2 : OK
No.3 : OK
No.4 : api_base_DataFeedDesc
cause DataFeedDesc was using on 1.8, and in https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/paddle_api_mapping_cn.html
there is no transfer for DataFeedDesc.
So I just change it to api_paddle_DataFeedDesc
No.5 : api_base_LoDTensor_en -> api_paddle_Tensor
Reason:
paddle/base/core.py line:279
```shell
libpaddle.LoDTensor = libpaddle.Tensor
```

### CODE REVIEW

File: python/paddle/base/data_feed_desc.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def __init__(self, proto_file):
 
     def set_batch_size(self, batch_size):
         r"""
-        Set :attr:`batch_size` in :ref:`api_base_DataFeedDesc` . :attr:`batch_size` can be changed during training.
+        Set :attr:`batch_size` in :ref:`api_paddle_DataFeedDesc` . :attr:`batch_size` can be changed during training.
```
Comment (sunzhongkai588) [inline python/paddle/base/data_feed_desc.py]: `DataFeedDesc` is not a public API. So do not use reference
```suggestion
        Set :attr:`batch_size` in ``paddle.base.DataFeedDesc`` . :attr:`batch_size` can be changed during training.
```

Comment (enkilee) [reply python/paddle/base/data_feed_desc.py]: OK

**Comment by luotao1** [comment]
@enkilee No.2-No.3 的PR已经合入了，可以去掉这部分修改
- #58861

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Pull request merged**